rest/route/github.go

```diff
@@ -118,6 +121,9 @@ func (gh *githubHookApi) shouldSkipWebhook(ctx context.Context, owner, repo stri
 }
 
 func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
+	ctx, cancel := context.WithTimeout(context.Background(), githubWebhookTimeout)
```

Comment:
Can we add a comment for future readers about the rationale for ignoring the request timeout (i.e. GitHub occasionally aborts requests early for no obvious reason)?